.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```

Comment:
missing newline.


BTW, would you mind adding also an `.editorconfig`?

```
root = true

[*]
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
indent_style = space
indent_size = 2

[*.md]
indent_size = 4
```

Reply:
@paulodiovani updated the files, have a look